src/state-manager/index.test.js

```diff
@@ -0,0 +1,34 @@
+import StateManager from './index';
+
+describe('StateManager', () => {
+  const stateManager = StateManager('url');
+  window.history.pushState({}, '', '?step=two');
+
+  test('can update browser history', () => {
+    stateManager.updateHistory('two', 'three');
+    expect(window.location.href).toContain('three');
+  });
+
+  test('can get active step', () => {
+    const activeStep = stateManager.getActiveStep();
+    expect(activeStep).toBe('three');
+  });
+});
+
+describe('StateManager disabled', () => {
+  test('does not init', () => {
+    expect(() => {
+      // eslint-disable-next-line no-unused-vars
+      const stateManager = StateManager();
```

Comment:
You can remove the eslint-disable-rule if you remove the variable. 

```suggestion
      StateManager();
```

Btw uppercase function names are [conventionally reserved for constructors](https://stackoverflow.com/questions/1564398/javascript-method-naming-lowercase-vs-uppercase).


```suggestion
      stateManager();
```

Reply:
Fixed and force pushed.